lsp/syntaxes/civet.json

```diff
@@ -167,7 +167,22 @@
       ]
     },
     {
-      "match": "\\b(?<![\\.\\$])(break|by|case|catch|continue|else|finally|for|in|of|if|return|switch|then|throw|try|unless|when|while|until|loop|do|export|import|default|from|as|yield|async|await|(?<=for)\\s+own)(?!\\s*:)\\b",
+      "name": "meta.import.civet",
+      "match": "(?:(import)\\s+(type)?\\s*)?(\\{(?:.|\\n)*\\}(?=\\s+from)|\\w+)\\s+(from)\\s+(\\S+)",
+      "captures": {
+        "1": { "name": "keyword.control.import.civet" },
+        "2": { "name": "keyword.control.type.civet" },
+        "3": {
+          "patterns": [
+            {"include": "source.civet"}
+          ]
+        },
+        "4": { "name": "keyword.control.from.civet" },
+        "5": { "name": "string.unquoted.module-reference.civet" }
+      }
+    },
+    {
+      "match": "\\b(?<![\\.\\$])(break|by|case|catch|continue|else|finally|for|in|of|if|return|switch|then|throw|try|unless|when|while|until|loop|do|export|import|default|from|as|yield|async|await|with|(?<=for)\\s+own)(?!\\s*:)\\b",
```

Comment:
Not related to this PR, but while I look at this, we have a bunch more `for` keywords, such as `each` and the reducers `sum` etc.